helm/templates/deployment.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sql-exporter.fullname" . }}
+  labels:
+    app: {{ include "sql-exporter.fullname" . }}
+    chart: {{ template "sql-exporter.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    {{- include "sql-exporter.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "sql-exporter.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        rollme: {{ randAlphaNum 5 | quote }}
```

Comment:
@allanger I think, in solutions like `ArgoCD` or `FluxCD` this line might cause constant resource drifts. I understand the intention, but I'm wondering if we can make it work on the `checksums` rather than random sequences. Otherwise, to drop the line and let it be populated with user-provided values?

Reply:
Ah, again, wasn't added by, copy-paste, I've overlooked. Let me check the whole chart one more time, I'll pay more attention to all the details. 
PS: I'm sorry, should've had a closer look before suggesting the PR, to get rid of all the strange stuff

Reply:
No worries, I also review file by file in my free time in the evening, so it goes slowly. We're not in a rush, though. 👍

Reply:
I checked the whole chart again and fixed some stuff. Also, I've found out that ServiceMonitor was broken, so I've fixed it and added a test. That's why Values.tests are back, this tests is supposed to be triggered only in CI, so I'm enabling it in Values